BitwardenShared/UI/Platform/Settings/Settings/AccountSecurity/DeleteAccount/DeleteAccountState.swift

```diff
@@ -2,4 +2,41 @@
 
 /// An object that defines the current state of a `DeleteAccountView`.
 ///
-struct DeleteAccountState: Equatable {}
+struct DeleteAccountState: Equatable {
+    // MARK: Properties
+
+    /// A short description of the view.
+    var description: String {
+        return if shouldPreventUserFromDeletingAccount {
+            Localizations.cannotDeleteAccount
+        } else {
+            Localizations.deletingYourAccountIsPermanent
+        }
+    }
+
+    /// A detailed description of the view.
+    var longDescription: String {
+        return if shouldPreventUserFromDeletingAccount {
+            Localizations.cannotDeleteAccountDescriptionLong
+        } else {
+            Localizations.deleteAccountExplanation
+        }
+    }
```

Comment:
⛏️ I'd change this to `title` and `description` as that's how it looks on the screen I think.